d2xx_linux_arm7hf.go

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+//go:build cgo && !no_d2xx
+// +build cgo,!no_d2xx
+
+package d2xx
+
+// TODO(maruel): https://github.com/golang/go/issues/7211 would help target the
```

Comment:
I don't think it'll ever get used, see the TODO. So you can remove the armv7 and armv8. Or we could deprecate support for armv6 but I would do this separately.

Reply:
Got it

Reply:
Please delete libftd2xx_linux_arm7hf_v1.4.27.a and libftd2xx_linux_arm8hf_v1.4.27.a as they are not used at the moment.

You may want to update https://github.com/periph/d2xx/blob/main/d2xx_linux_arm64.go to point to arm8hf then, and delete the current libftd2xx_linux_arm64_v1.4.24.a static library?

It's fairly possible I had renamed it? It's probably a good idea to use arm8hf instead of arm64 for the static library name. I don't have a strong stance on the name, as long as there's only one copy.

Reply:
> Please delete libftd2xx_linux_arm7hf_v1.4.27.a and libftd2xx_linux_arm8hf_v1.4.27.a as they are not used at the moment.
> 

In that case we should keep libftd2xx_linux_arm8hf_v1.4.27.a, as we are pointing https://github.com/periph/d2xx/blob/main/d2xx_linux_arm64.go to arm8hf ?

Please confirm

Reply:
Yep, and delete the previous libftd2xx_linux_arm64_v1.4.24.a one.